tests/returntypes/test_jax_qnode_new.py

```diff
@@ -2190,3 +2190,1194 @@ def circuit(x):
+            assert jac[1][0].shape == (4,)
+            assert isinstance(jac[1][1], jax.numpy.ndarray)
+            assert jac[1][1].shape == (4,)
+        else:
+            num_copies = sum(
+                [1 for x in shots if isinstance(x, int)]
+                + [x[1] for x in shots if isinstance(x, tuple)]
+            )
+            assert len(jac) == num_copies
+            for j in jac:
+                assert isinstance(j, tuple)
+                assert len(j) == 2
+
+                assert isinstance(j[0], tuple)
+                assert len(j[0]) == 2
+                assert isinstance(j[0][0], jax.numpy.ndarray)
+                assert j[0][0].shape == ()
+                assert isinstance(j[0][1], jax.numpy.ndarray)
+                assert j[0][1].shape == ()
+
+                assert isinstance(j[1], tuple)
+                assert len(j[1]) == 2
+                assert isinstance(j[1][0], jax.numpy.ndarray)
+                assert j[1][0].shape == (4,)
+                assert isinstance(j[1][1], jax.numpy.ndarray)
+                assert j[1][1].shape == (4,)
+
+    @pytest.mark.parametrize("jacobian", jacobian_fn)
+    def test_jacobian_multiple_measurement_multiple_param_array(
+        self, dev_name, diff_method, gradient_kwargs, shots, jacobian
+    ):
+        """The jacobian of multiple measurements with a multiple params array return a single array."""
+        dev = qml.device(dev_name, wires=2, shots=shots)
+
+        @qnode(dev, interface="jax", diff_method=diff_method, **gradient_kwargs)
+        def circuit(a):
+            qml.RY(a[0], wires=0)
+            qml.RX(a[1], wires=0)
+            return qml.expval(qml.PauliZ(0)), qml.probs(wires=[0, 1])
+
+        a = jax.numpy.array([0.1, 0.2])
+
+        jac = jacobian(circuit)(a)
+
+        if isinstance(shots, int):
+            assert isinstance(jac, tuple)
+            assert len(jac) == 2  # measurements
+
+            assert isinstance(jac[0], jax.numpy.ndarray)
+            assert jac[0].shape == (2,)
+
+            assert isinstance(jac[1], jax.numpy.ndarray)
+            assert jac[1].shape == (4, 2)
+        else:
+            num_copies = sum(
+                [1 for x in shots if isinstance(x, int)]
+                + [x[1] for x in shots if isinstance(x, tuple)]
+            )
+            assert len(jac) == num_copies
+            for j in jac:
+                assert isinstance(j, tuple)
+                assert len(j) == 2  # measurements
+
+                assert isinstance(j[0], jax.numpy.ndarray)
+                assert j[0].shape == (2,)
+
+                assert isinstance(j[1], jax.numpy.ndarray)
+                assert j[1].shape == (4, 2)
+
+    def test_hessian_expval_multiple_params(self, dev_name, diff_method, gradient_kwargs, shots):
+        """The hessian of single a measurement with multiple params return a tuple of arrays."""
+        import jax
+        from jax.config import config
+
+        config.update("jax_enable_x64", True)
```

Comment:
Is this required specifically in this test case? (recurring for some of the following test cases too)

Reply:
Removed everywhere.